giraffe/src/transforms/line.ts

```diff
@@ -1,13 +1,30 @@
-import {Table, LineLayerSpec, LineData} from '../types'
+import {Table, LineLayerSpec, LineData, LinePosition} from '../types'
 import {FILL} from '../constants/columnKeys'
 import {createGroupIDColumn, getNominalColorScale} from './'
 
+const getPrevCumulativeDomainValue = (
+  lineData: LineData,
+  groupID: number,
+  domainLabel: string
```

Comment:
I'm surprised this was not typed previously. Would you be willing to try typing 'xs' and 'ys' as an enum?

Reply:
Sure thing. Will add that to the next push.